charts/prometheus-yet-another-cloudwatch-exporter/README.md

```diff
@@ -0,0 +1,18 @@
+# yet-another-cloudwatch-exporter
+
+![Version: 0.38.0](https://img.shields.io/badge/Version-0.38.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.61.2](https://img.shields.io/badge/AppVersion-v0.61.2-informational?style=flat-square)
+
+Yace - Yet Another CloudWatch Exporter
+
+**Homepage:** <https://github.com/nerdswords/helm-charts>
+
+## Installation
+
+```sh
+helm repo add nerdswords https://nerdswords.github.io/helm-charts
+helm install nerdswords/yet-another-cloudwatch-exporter
```

Comment:
Please update the URLs here. You could also favorite the OCI chart package, like `oci://ghcr.io/prometheus-community/charts/kube-prometheus-stack `